qiskit_metal/analyses/quantization/sim_eigenmode.py

```diff
@@ -127,14 +127,17 @@ def run_sim(  # pylint: disable=arguments-differ
         if not self.renderer_initialized:
             self._initialize_renderer()
 
-        renderer_design_name = self._render(name=name,
-                                            solution_type='eigenmode',
-                                            selection=components,
-                                            open_pins=open_terminations,
-                                            port_list=port_list,
-                                            jj_to_port=jj_to_port,
-                                            ignored_jjs=ignored_jjs,
-                                            box_plus_buffer=box_plus_buffer)
+        vars_to_initialize = self.setup.sim.vars
```

Comment:
this will need to change to `self.sim.setup.vars` after #642 is merged. Unless we merge this one first, then we need to change it inside #642 before merging.

Reply:
Do you mean `self.setup.vars`?

Reply:
yes

Reply:
looks good, also the other catches of the same around the code.